src/luascript.cpp

```diff
@@ -4709,7 +4710,7 @@ int LuaScriptInterface::luaGameCreateMonster(lua_State* L)
 
 int LuaScriptInterface::luaGameCreateNpc(lua_State* L)
 {
-	// Game.createNpc(npcName, position[, extended = false[, force = false]])
+	// Game.createNpc(npcName, position[, extended = false[, force = false[, magicEffect = CONST_ME_TELEPORT]]])
```

Comment:
I assume this is backwards compatible, because `bool` isn't a valid number and `CONST_ME_TELEPORT` would be used?

Reply:
It doesn't affect `Game.createNpc` & `Game.createMonster` since it's a new parameter whereas Player.setGhostMode has its `showEffect` (boolean) replaced with an integer input. It should default to `CONST_ME_TELEPORT` to my knowledge.